SparkPost/2web2ui
# FE-1376 - Resolves Upgrade high severity vulnerabilities - immer and set-value

[FE-1376](https://sparkpost.atlassian.net/browse/FE-1376)

### What Changed

- Manually installs safe versions of `immer` and `set-value` that are transient dependencies
- Updates `lint-staged` which referenced a vulnerable version of `set-value` - and updated our implementation according to the migration notes

### How To Test

- Run `npm i` on this branch
- Ensure the app runs locally as expected
- Make a linting error (add a `console.log`, for example) and try and commit the file - ensure the error is caught and no commit is made

### To Do
